server/src/db/seed/user.seed.ts

```diff
@@ -0,0 +1,68 @@
+import { type DrizzleDb } from '../index';
+import { hashPassword } from '~/utils/password';
+import { userTable } from '~/domain/user/user.db';
+
+export type User = {
+  id: string;
+  name: string;
+};
+
+export type UserSeed = {
+  admin: User;
+  manager: User;
+  viewer: User;
+  user1: User;
+  user2: User;
+  user3: User;
+};
+
+export async function seed(db: DrizzleDb): Promise<UserSeed> {
+  const admin = await createUser(db, {
+    email: 'admin@test',
```

Comment:
@keskiju you forgot the `.com` in your latest changes so it does not reflect the output of `taito info`

Reply:
I originally didn't want to name the test users as "Admin|Manager|Viewer something" as my idea was that different users can be admin on different organisations so it wouldn't make sense to hard code the role in the user's name.

Reply:
Fixed the missing `.com`.